api/src/api/userRouter.test.ts

```diff
@@ -380,139 +380,149 @@ describe("userRouter", () => {
         mockJwt.decode.mockReturnValue(cognitoPayload({ id: "123" }));
       });
 
-      it("does not update business name search if businessNameAvailability is undefined", async () => {
-        const userData = generateUserDataForBusiness(
-          generateBusiness({
-            formationData: generateFormationData({ businessNameAvailability: undefined }),
-          })
-        );
-        stubUserDataClient.get.mockResolvedValue(userData);
+      describe("when businessPersona is 'STARTING'", () => {
+        it("does not update business name search if businessNameAvailability is undefined", async () => {
+          const userData = generateUserDataForBusiness(
+            generateBusiness({
+              profileData: generateProfileData({ businessPersona: "STARTING" }),
```

Comment:
[dust] It looks like all of these are generating `profileData` with `businessPersona: "STARTING"`. How do you feel about creating a function at the `describe` level that sets this?